.github/workflows/stack-integration_tests.yml

```diff
@@ -653,3 +653,19 @@ jobs:
           docker rm `docker ps -aq` --force || true
           tox -e stack.test.course
           docker rm `docker ps -aq` --force || true
+
+      - name: Job Report Status
+        if: always()
+        uses: ravsamhq/notify-slack-action@v1
+        with:
+          status: ${{ job.status }}
+          notify_when: 'failure'
+          notification_title: '{workflow} has {status_message}'
+          message_format: '{emoji} *{workflow}* {status_message} in <{repo_url}|{repo}>'
```

Comment:
Maybe we can add the branch_url so we have link to the branch itself.

Reply:
@shubham3121 the {branch_url} doesn't seem to generate a correct context  URL After all, there are workarounds but will take some hacking. For the moment, I have manually constructed and added the PR URL instead at the footer and this should work for now.

Reply:
Great !! 👍🏾